config/routes.rb

```diff
@@ -2,6 +2,7 @@
 
   root 'homes#index'
   devise_for :users, path: 'accounts'
+  match 'users/:id' => 'users#destroy', :via => :delete, :as => :admin_destroy_user
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.